docs/src/enterprise/getstarted/migrate-from-fluffy.md

```diff
@@ -0,0 +1,299 @@
+    
+        ```yaml
+        # lakefs.yaml
+        auth:
+          oidc:
+            friendly_name_claim_name: "name"
+            persist_friendly_name: true
+            default_initial_groups: ["Developers"]
+          ui_config:
+            login_url: /oidc/login
+            logout_url: /oidc/logout
+            login_cookie_names:
+            - internal_auth_session
+            - oidc_auth_session
+        ```
+
+    === "lakeFS Enterprise (new)"
+        
+        ```yaml
+        # lakefs.yaml
+        auth:
+          logout_redirect_url:  https://oidc-provider-url.com/logout/url
+          ui_config:
+            login_url: /oidc/login
+            logout_url: /oidc/logout
+            login_cookie_names:
+              - internal_auth_session
+              - oidc_auth_session
+          oidc:
+            friendly_name_claim_name: "nickname"
+            default_initial_groups: ["Admins"]
+          providers:
+            oidc:
+              # enabled: true  # This field was dropped!
+              post_login_redirect_url: / # This field was moved here!
+              url: https://oidc-provider-url.com/
+              client_id: <oidc-client-id>
+              client_secret: <oidc-client-secret>
+              callback_base_url: https://lakefs.company.com
+              logout_client_id_query_parameter: client_id
+              logout_endpoint_query_parameters:
+                - returnTo
+                - http://lakefs.company.com/oidc/login
+        ```
+
+
+!!! note "LDAP"
+   
+    === "lakeFS + Fluffy (old)"
+        
+        ```yaml
+        # fluffy.yaml
+        auth:
+          post_login_redirect_url: /
+          ldap: 
+            server_endpoint: 'ldaps://ldap.company.com:636'
+            bind_dn: uid=<bind-user-name>,ou=<some-ou>,o=<org-id>,dc=<company>,dc=com
+            bind_password: '<ldap password>'
+            username_attribute: uid
+            user_base_dn: ou=<some-ou>,o=<org-id>,dc=<company>,dc=com
+            user_filter: (objectClass=inetOrgPerson)
+            connection_timeout_seconds: 15
+            request_timeout_seconds: 7
+	    ```
+        ```yaml
+    	# lakefs.yaml
+        auth:
+          remote_authenticator:
+            enabled: true
+            endpoint: http://<Fluffy URL>:<Fluffy http port>/api/v1/ldap/login
+            default_user_group: "Developers" # Value needs to correspond with an existing group in lakeFS
+          ui_config:
+            logout_url: /logout
+            login_cookie_names:
+            - internal_auth_session
+        ```
+    
+    === "lakeFS Enterprise (new)"
+        
+        ```yaml
+        # lakefs.yaml
+        auth:
+          ui_config:
+            logout_url: /logout
+            login_cookie_names:
+              - internal_auth_session
+          providers:
+            ldap:
+              server_endpoint: ldaps://ldap.jumpcloud.com:636
+              bind_dn: uid=bind_app,ou=Users,o=63eb8e248d5156c875be4124,dc=jumpcloud,dc=com
```

Comment:
All values should be equal between previous and old.
 This is specific value to a test env. 
Keep the old example values please. in all config (uid=<bind-user-name>,ou=<some-ou>,o=<org-id>,dc=<company>,dc=com)